egs/wsj/s5/steps/nnet3/lstm/train.sh

```diff
@@ -560,10 +560,12 @@ while [ $x -lt $num_iters ]; do
       cur_num_hidden_layers=$[1+$x/$add_layers_period]
       config=$dir/configs/layer$cur_num_hidden_layers.config
       raw="nnet3-am-copy --raw=true --learning-rate=$this_learning_rate $dir/$x.mdl - | nnet3-init --srand=$x - $config - |"
+      cache_io_opts="--read-cache=$dir/cache.$x"
```

Comment:
You should be setting this cache_io_opts to "" and the other one to this value- you have switched them.